core/check/ifEmpty.go

```diff
@@ -0,0 +1,10 @@
+package check
+
+import (
+	"strings"
+)
+
+// IfEmpty tests if the provided string is technically empty
+func IfEmpty(s string) bool {
+	return strings.TrimSpace(s) == ""
```

Comment:
is this enough? What happens on this string: "\t "

Reply:
changed with `IfHrp()`